cdhweb/blog/models.py

```diff
@@ -54,40 +58,106 @@ class BlogPostTag(TaggedItemBase):
     )
 
 
-class BlogPost(BasePage, ClusterableModel, PagePreviewDescriptionMixin):
+class BlogPost(BasePage, ClusterableModel):
     """A Blog post, implemented as a Wagtail page."""
 
-    featured_image = models.ForeignKey(
+    template = "blog/blog_post.html"
+
+    description = RichTextField(
+        max_length=200,
+        blank=True,
+        null=True,
+        features=["bold", "italic"],
+        verbose_name="Blog description",
+        help_text="Short introduction to the page, aim for max two clear sentences (max. 200 chars).",
+    )
+
+    article_date = models.DateTimeField(
+        verbose_name="Published date",
+        default=localdate,
+        help_text=("Published date for blog post"),
+    )
+
+    image = models.ForeignKey(
         "wagtailimages.image",
         null=True,
         blank=True,
         on_delete=models.SET_NULL,
         related_name="+",
         help_text="Appears on the homepage carousel when post is featured.",
     )
+    caption = RichTextField(
+        features=[
+            "italic",
+            "bold",
+            "link",
+        ],
+        help_text="A short caption for the image.",
+        blank=True,
+        max_length=180,
+    )
+
+    credit = RichTextField(
+        features=[
+            "italic",
+            "bold",
+            "link",
+        ],
+        help_text="A credit line or attribution for the image.",
+        blank=True,
+        max_length=80,
+    )
+
+    alt_text = models.CharField(
+        help_text="Describe the image for screen readers",
+        blank=True,
+        max_length=80,
+    )
     tags = ClusterTaggableManager(through=BlogPostTag, blank=True)
     featured = models.BooleanField(
         default=False, help_text="Show the post in the carousel on the homepage."
     )
     people = models.ManyToManyField(Person, through="blog.Author", related_name="posts")
 
     # can only be created underneath special link page
-    parent_page_types = ["blog.BlogLinkPage"]
+    parent_page_types = ["blog.BlogLinkPageArchived", "blog.BlogLandingPage"]
```

Comment:
You could take the archived one out of here, if you don't want them to add any blogs to the old one (up to you whether that's the right thing or not though)

Reply:
I was planning on leaving these in until I move the pages to the new landing page type